system_setup/models/system_setup.py

```diff
@@ -61,6 +62,23 @@ def __init__(self, root, hub, install_model_names,
         self.wslconfbase = WSLConfigurationBaseModel()
         self.wslconfadvanced = WSLConfigurationAdvancedModel()
 
+        if prefillInfo:
```

Comment:
If the controller is handling this we don't also need it in the model, right?

Reply:
That was embarrassing... 😳
That piece of code was supposed to be removed. My fault while experimenting between local branches. Ended up leaving some dead code.